cluster/ci/config/prow/config.yaml

```diff
@@ -3492,6 +3492,10 @@ periodics:
       master: ci.openshift.redhat.com
   interval: 24h
   run_after_success:
+  - name: azure_build_node_image_centos_310
```

Comment:
Hrm, @jim-minter removed these jobs in https://github.com/openshift/aos-cd-jobs/pull/1469

Reply:
@kargakis, the ones he removed were for base image builds.  We still need node images.

Reply:
sorry, didn't look close - is there an aos-cd-jobs pr that adds the node 3.10 jobs?

Reply:
@kargakis, looks like peter and clayton beat us to it: 

https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/azure_build_node_image_centos_310.yml